tests/OSPSuite.Core.IntegrationTests/DiffBuilders/ParameterDiffBuilderSpecs.cs

```diff
@@ -191,7 +191,7 @@ public void should_have_some_differences()
          _report.Count.ShouldBeEqualTo(1);
          var propertyDiffItem = _report[0].DowncastTo<PropertyValueDiffItem>();
          propertyDiffItem.FormattedValue1.ShouldBeEqualTo("1.000 m");
-         propertyDiffItem.FormattedValue2.ShouldBeEqualTo("1.000 mm");
+         propertyDiffItem.FormattedValue2.ShouldBeEqualTo("1.000E-3 m");
```

Comment:
Let's make this a separate observation with a name that reflects the behavior that we are expecting.

This is changed because the first item in the comparison is in meters right?

Please make the observation name explicit for that behavior

Reply:
Yes, that's correct, the first item is leading as far as units are concerned. 

I added a test to check the two different scenario's separate